app_dart/lib/src/model/github/create_event.dart

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// CreateEvent model that is generated from GitHub Create event webhook.
+///
+/// See more:
+///  * https://docs.github.com/en/developers/webhooks-and-events/events/github-event-types#createevent
+import 'package:github/github.dart' show Repository, User;
+import 'package:github/hooks.dart' show HookEvent;
+import 'package:json_annotation/json_annotation.dart';
+
+part 'create_event.g.dart';
+
+@JsonSerializable(fieldRename: FieldRename.snake)
+class CreateEvent extends HookEvent {
```

Comment:
This should get upstreamed in `package:github` first. Feel free to add me as a reviewer there

Reply:
sure, raised pr at https://github.com/SpinlockLabs/github.dart/pull/304

Reply:
SGTM